### ISSUE TITLE
Site should be WCAG AA compliant

### ISSUE THREAD

**Issue by will-sonnex** [Volunteer-Victoria/volunteer-victoria-community#70]

**Comment by stabback** [comment]
Note - this is an ongoing task.  This issue is a marker to get us there for now, but all future improvements should also be WCAG AA compliant.

**Comment by stabback** [comment]
Fixed in https://github.com/Volunteer-Victoria/volunteer-victoria-community/commit/eed5419040f3788a1a5c0189985addd5434f76a4
